xlators/mgmt/glusterd/src/glusterd-utils.c

```diff
@@ -14970,6 +15021,35 @@ glusterd_check_brick_order(dict_t *dict, char *err_str, int32_t type,
         }
         ++j;
     }
+
+    /* Check for bad brick order for brick list present already in the volume
+     * with the bricks provided in add-brick command.
+     * pre_list = [pn1, pn2, pn3]
+     * ai_list = [pn4, pn1]
+     * Failed as pn1 is present in new_list for replica subvols
+     */
+    ai_list_tmp2 = NULL;
+    if (flag) {
+        i = 0;
+        j = 0;
```

Comment:
Can we remove this? It seems like `j` isn't being put to use after this stage in the function. So assignment operation here seems to be of no use.

Reply:
Done